framework/dbt/basic/README.md

```diff
@@ -0,0 +1,15 @@
+Welcome to your new dbt project!
+
+### Using the starter project
+
+Try running the following commands:
+- dbt run
```

Comment:
```suggestion
- `dbt run`
```

Reply:
Thanks. `dbt init` created the README file that way.